README.md

```diff
@@ -26,9 +26,9 @@ If this is your very first Lando project, make sure that your system trusts the
 
 ## Usage
 
-* WordPress core contributions can happen either via the `public/core-git` directory (if you prefer to use git) or via the `public/trunk-svn` directory (if you prefer to use SVN).
+* WordPress core contributions can happen either via the `public/core-dev` directory (if you prefer to use git) or via the `public/trunk-svn` directory (if you prefer to use SVN).
```

Comment:
Maybe better:
WordPress core contributions can happen via the `public/core-dev` directory which supports both git and SVN.

Reply:
I think we should also add a few sub bullet-points to this one, explaining how you can sync the two with examples such as `git svn-up master` or `git svn-up 5.0`. We don't need to otherwise dive in how to use `git` or `SVN` in the readme since this should be a given, but this custom alias needs to be mentioned, and we should probably also say that it's a shortcut to running our custom script.

Reply:
Done in ea74377